sdcflows/interfaces/utils.py

```diff
@@ -297,6 +297,33 @@ def _run_interface(self, runtime):
         return runtime
 
 
+class _PositiveDirectionCosinesInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc="input image")
+
+
+class _PositiveDirectionCosinesOutputSpec(TraitedSpec):
+    out_file = File(exists=True)
```

Comment:
```suggestion
    out_file = File()
```

Reply:
Sure, although this has no effect, as we were discussing in niworkflows.